internal/config/default_config.go

```diff
@@ -29,6 +29,14 @@ var (
 				Name:      "secrets.v1.",
 				PublishTo: "meshery.meshsync.core",
 			},
+			{
+				Name:      "persistentvolumes.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			{
+				Name:      "persistentvolumeclaims.v1.",
```

Comment:
What negative ramifications are involved in removing the “v1“? Any? If not, please do.